app/controllers/dashboard_controller.rb

```diff
@@ -97,4 +76,12 @@ def host_warning
                     "It is advised to access velum via one of the registered hosts " \
                     "#{accessible_hosts.join(" or ")}"
   end
+
+  # Pollutes the instance variables of this object by copying over all the
+  # variables of the parameter
+  def pollute_variables!(other)
+    other.instance_variables.each do |var|
+      instance_variable_set(var, other.instance_variable_get(var))
```

Comment:
could you clarify a bit more what this is supposed to do? to me it looks like it's reading the instance variables and sets them again to the same value. is this to "bring them over" from the lib to the controller namespace? i dont know if that's needed, or if they're accessible during runtime anyways without setting them again. i'd add a clearer comment because this is meta hacking ;-)

the parameter name `other` is also a bit unclear to me still, maybe it's a bit too generic

Reply:
`var` might also need to be symbolized, looking at https://apidock.com/ruby/Object/instance_variable_set but i havent tested it yet

Reply:
`object.instance_variables` already returns the symbolized form, so it should be fine:

```ruby
[1] pry(main)> class A
[1] pry(main)*   def initialize
[1] pry(main)*     @a = "a"
[1] pry(main)*     @b = "b"
[1] pry(main)*   end  
[1] pry(main)* end  
=> :initialize
[2] pry(main)> a.instance_variables
=> [:@a, :@b]
```

And I really dislike the `meta-programming` copying - but there are some tests that really expect the templates to render with only the variable names. I added this in a comment, but maybe it is not worth to hack it this way, just for those tests and we could remove them?

Then I would just assign `@autoyast = ...` and update the template to render the variables with an `autoyast` prefix.